Cargo.toml

```diff
@@ -103,6 +103,9 @@ revm = { version = "16.0.0", default-features = false }
 # K/V database
 rocksdb = { version = "0.22.0", default-features = false }
 
+# ZKVM
+risc0-zkvm = { version = "1.2.3", features = ["heap-embedded-alloc", "unstable"] }
```

Comment:
think in general its more customary to turn off all features in the workspace cargo.toml and only turn on the ones that you need per crate. Although here I'm guessing this risc0-zkvm is only used by the proof crate right?